src/main/java/com/WWI16AMA/backend_api/Account/Account.java

```diff
@@ -11,7 +11,7 @@
 public class Account {
 
     @Id
-    @GenericGenerator(name = "5-digit-Id", strategy = "com.WWI16AMA.backend_api.Account.CustomGenerator.AccountIdGenerator")
+    @GenericGenerator(name = "5-digit-Id", strategy = "com.WWI16AMA.backend_api.CustomGenerator.AccountIdGenerator")
```

Comment:
Die Auslagerung hatte ich damals nicht gemacht, weil wir halt nur für Member und Account die zufälligen ID's haben. Würde ein weiterer Generator dazukommen, würde ich eh überlegen, die irgendwie so zu refactoren, dass das alles smoother is und vllt ein Generator vom anderen erbt oder so. Halt "schön". Aber ich seh net so krass den Sinn, die beiden Generatoren in ein extra package zu klatschen

Reply:
Sollten wir in gelaber einmal ansprechen, ich finds sinnvoll dass auszulagern.....